### PR TITLE
Changelog for r115

### PR DESCRIPTION
**Pull request by esbenbach**
Included is the changelog/releasenotes for r115.

### CODE REVIEW

File: source/changelog/r115.inc [hunk 1]
```diff
@@ -0,0 +1,123 @@
+.. _changelog-r115:
+
+************************
+Release August 10th 2023
+************************
+
+Summer vacations are almost over for everyone in the dev team, and a new release is ready, roughly 2 months since the previous release.
+This release is a mixed bag of fixes, small incremental improvements and a single new feature in preview.
+
+We a still trying out the new format for the changelog, following a type of change convention outlined on https://keepachangelog.com/
```
Comment (BrynjarS) [inline source/changelog/r115.inc:10]: Typo: "We a still" -> "We are still"

File: source/changelog/r115.inc [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+* Merchants can now configure Self-Service behaviour for Subscription Cancellations.
+* Added an indicator showing if there are Invoices on a given subscriber.
+* **[Preview]** Support for listing the sum of not-invoice Billing Account transactions on the ledger summary.
+* **[Preview]** Support for configuring Domain Authorization on Subscriptions.
+
+
+Fixed
+-----
+
+* An issue with the wrong value being show for Reminder Fees.
```
Comment (BrynjarS) [inline source/changelog/r115.inc:86]: Typo: show -> shown

File: source/changelog/r115.inc [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+* An issue where product with an adjusted price would be displayed as expired.
+* Tenant selection should again be persisted between session on the same browser.
+* An invalid Subscription Plan would sometimes be displayed when doing refunds.
+* An issue where it was not possible to remove a planned cancellation an already renewed Subscription.
```
Comment (BrynjarS) [inline source/changelog/r115.inc:90]: Typo? "cancellation an already" -> "cancellation of an already"

**Review by BrynjarS — APPROVED** [review]

File: source/changelog/r115.inc [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+===============
+The following new features and improvements are now available in the merchant client.
+
+Added
+-----
+
+* **[Preview]** Support for listing the sum of not-invoice Billing Account transactions on the ledger summary.
+* **[Preview]** Support for configuring Domain Authorization on Subscriptions.
+* Merchants can now configure Self-Service behaviour for Subscription Cancellations.
+* Added an indicator showing if there are Invoices on a given subscriber.
```
Comment (KleisKlasKluss) [inline source/changelog/r115.inc:78]: Notes, not Invoices

File: source/changelog/r115.inc [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+
+Fixed
+-----
+
+* An issue with the wrong value being show for Reminder Fees.
```
Comment (KleisKlasKluss) [inline source/changelog/r115.inc:86]: It was the Tax amount in Invoices (just the overview) that were incorrect, for Reminders with Fees AND tax calculation for invoice fee

File: source/changelog/r115.inc [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+* An issue where product with an adjusted price would be displayed as expired.
+* Tenant selection should again be persisted between session on the same browser.
+* An invalid Subscription Plan would sometimes be displayed when doing refunds.
+* An issue where it was not possible to remove a planned cancellation an already renewed Subscription.
+* Removed an invalid button from |eFaktura| payment agreement view.
+* An issue with custom grid sorting for |eFaktura| transactions being ignored.
```
Comment (KleisKlasKluss) [inline source/changelog/r115.inc:92]: eFaktura and AvtaleGiro

**Pull request merged**